test/functional/rpc_fundrawtransaction.py

```diff
@@ -480,22 +490,39 @@ def test_spend_2of2(self):
         oldBalance = self.nodes[1].getbalance()
         inputs = []
         outputs = {self.nodes[1].getnewaddress():1.1}
-        rawtx = self.nodes[2].createrawtransaction(inputs, outputs)
```

Comment:
In e3dc50344a57b926373bcc51aa2e213a682051be it seems problematic to replace  `createrawtransaction` and `fundrawtransaction` with PSBT methods in `rpc_fundrawtransaction.py`.

Reply:
I think it's fine in some instances (like this one) because the funding code is the same and the goal is to check that the funding works.

Reply:
Do we still sufficiently test the `createrawtransaction` and `fundrawtransaction` RPC elsewhere? Those are rather ancient, with all cool kids using PSBT it might be a while before we see a regression.

Reply:
IMO we do. `createrawtransaction` and `fundrawtransaction` are still being tested in `rpc_fundrawtransaction.py` as well is in other tests. Additionally, `fundrawtransaction` basically only calls `FundTransaction` after a bit of options parsing, and `walletcreatefundedpsbt` calls this function as well, right after transaction creation.